molo/profiles/views.py

```diff
@@ -105,3 +112,141 @@ def form_valid(self, form):
         )
         return render(self.request, self.template_name,
                       {'form': form})
+
+
+class ForgotPasswordView(FormView):
+    form_class = forms.ForgotPasswordForm
+    template_name = "profiles/forgot_password.html"
+
+    # Retrieve the required number of security questions (at random)
+    num_security_questions = settings.SECURITY_QUESTION_COUNT \
```

Comment:
Can we get the `num_security_questions` from `UserProfilesSettings` in wagtail, instead of settings
http://docs.wagtail.io/en/v1.4.1/reference/contrib/settings.html
you can do something similar to https://github.com/praekelt/molo.profiles/blob/develop/molo/profiles/forms.py#L48-L57